Packs/VMwareWorkspaceONEUEM/Integrations/VMwareWorkspaceONEUEM/test_data/get_device_404_no_records_found_message.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+
```

Comment:
This response is parsed in the integration? 
Why do we need this data?

Reply:
No @Shellyber, this response is not parsed in the integration. 
We're using this for Unit Testing when API returns 404, we get HTML content.